Kudu.Core/SiteExtensions/SiteExtensionManager.cs

```diff
@@ -50,6 +50,8 @@ public class SiteExtensionManager : ISiteExtensionManager
         private readonly IContinuousJobsManager _continuousJobManager;
         private readonly ITriggeredJobsManager _triggeredJobManager;
 
+        private static readonly string _toBeDeletedDirectoryPath = Path.Combine(System.Environment.ExpandEnvironmentVariables(@"%HOME%\data\SiteExtensions"), "tobedeleted");
```

Comment:
From MSDN:

[If you try to move a file across disk volumes and that file is in use, the file is copied to the destination, but it is not deleted from the source.](https://msdn.microsoft.com/en-us/library/system.io.file.move.aspx)